spec/services/user_invite_service_spec.rb

```diff
@@ -50,5 +52,51 @@
       expect(result).to have_role(Role::ORG_ADMIN, organization)
       expect(result).not_to have_role(Role::PARTNER, :any)
     end
+
+    it "should create the user without name with role" do
+      result = nil
+      expect {
+        result = described_class.invite(name: "",
```

Comment:
I suggest changing this to passing in no name at all (which many invites do I think):

```suggestion
        result = described_class.invite(
```